src/unix/bsd/netbsdlike/mod.rs

```diff
@@ -439,6 +439,34 @@ pub const MNT_NODEV: c_int = 0x00000010;
 pub const MNT_LOCAL: c_int = 0x00001000;
 pub const MNT_QUOTA: c_int = 0x00002000;
 
+// sys/ioccom.h in NetBSD and OpenBSD
+pub const IOCPARM_MASK: u32 = 0x1fff;
+
+pub const IOC_VOID: u32 = 0x20000000;
+pub const IOC_OUT: u32 = 0x40000000;
+pub const IOC_IN: u32 = 0x80000000;
+pub const IOC_INOUT: u32 = IOC_IN | IOC_IN;
```

Comment:
`bad IOC_INOUT value at byte 3: rust: 128 (0x80) != c 192 (0xc0)`

small typo. the value should be `IOC_IN | IOC_OUT`

Reply:
thanks a lot